.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: dart
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git -b stable --depth 1
+  - ./flutter/bin/flutter doctor
+  - ./flutter/bin/flutter packages get
+script:
+  - cd packages/golden_toolkit
+  - ../../flutter/bin/flutter analyze
+  - ../../flutter/bin/cache/dart-sdk/bin/dartfmt -n ./lib --set-exit-if-changed
+  - ../../flutter/bin/flutter test --coverage --coverage-path=lcov.info
+# after_success:
+#   - bash <(curl -s https://codecov.io/bash)
```

Comment:
```suggestion
```

Reply:
actually... better yet...

Reply:
ebay teams are already using codecov. Just going to turn it on 😄